### DISCUSSION TITLE
Can not set template in gradle

### DISCUSSION THREAD

**Discussion by LethalSnake1337** [rodnansol/spring-configuration-property-documenter#97] [Q&A]
When I try to set my path to my custom handlebar template, gradle fails to build with the following exception
**Illegal char <:> at index 3 ./C:\Users\.....**

I am adding the path as follows 
`template = file("src/main/resources/content.md") `

I also tried my absolute path it's the same problem.

Does anyone maybe know why this is happening? Why is ./ added to the path? I am on Windows.

Thank you in advance.

**Comment by nandorholozsnyak** [comment]
Hey  @LethalSnake1337 

Let me dig deeper and find this problem, unfortunately I'm on Linux, but I will try to do my best.

**Reply by LethalSnake1337** [reply]
That would be awesome thank you. If you need any help let me know.
